texts/epn_paper.md

```diff
@@ -0,0 +1,19 @@
+# Let's talk online
+
```

Comment:
Add your text directly below.

**Important:** before starting to write a paragraph, leave a comment in the paragraph title, so that others don't work on it as well.